ClassTranscribeDatabase/CommonUtils.cs

```diff
@@ -49,17 +52,57 @@ public static T BytesToMessage<T>(byte[] bytes)
             return JsonConvert.DeserializeObject<T>(Encoding.UTF8.GetString(bytes));
         }
 
+        private static int seed; // member var only for future debugging /replay options
+        private static Random random;
+        /// <summary>Ensures we have a well seeded RNG even if two processes or two threads  are started at the same time</summary>    
+        private static Random getSharedRandom()
+        {
+            if (random != null) 
+                return random; // Early return
+            // Use something external and trust worthy if it exists
+            const string randomStream = "/dev/urandom";
+            if (File.Exists(randomStream)) {
+                using (BinaryReader reader = new BinaryReader(File.Open(randomStream, FileMode.Open)))
+                {
+                    seed = reader.ReadInt32();
+                }
+            } else
+            {
+                // Fallback
+                seed = Guid.NewGuid().ToString().GetHashCode();
+            }
+            // TODO/REVIEW:Random ctor seed is limited to 32 bits!
+            // Eventually we will need something better
+            random = new Random(seed);
+            return random;
+        }
         public static string RandomString(int length)
         {
-            Random random = new Random();
-            const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
-            return new string(Enumerable.Repeat(chars, length)
-              .Select(s => s[random.Next(s.Length)]).ToArray());
+            // TODO/TOREVIEW: C# random is seeded with a time since 1970 at 100ns, so what if two
```

Comment:
Why not use [GUID](https://docs.microsoft.com/en-us/dotnet/api/system.guid.newguid?redirectedfrom=MSDN&view=netcore-3.1#System_Guid_NewGuid)

Reply:
Good idea. Added it to  https://github.com/classtranscribe/WebAPI/issues/90